### PR TITLE
Invisible recording transfers and Core camera coordinate system clarification

### PR DESCRIPTION
**Pull request by N-M-T**

### CODE REVIEW

File: src/invisible/user-guide/invisible-companion-app.md [hunk 1]
```diff
@@ -77,3 +80,20 @@ If more that one device is used and the data is required to be synchronised, mak
...
+
+Invisible recordings are uploaded to the cloud from the Invisible Companion App to ensure stability, robustness and security. If you still have the recordings on the Companion device, follow these instructions:
+
+1) Enable 'Cloud Upload' in the Companion App settings
+2) Wait until your recording/s are uploaded to Pupil Cloud. In the recording view, uploaded recordings will show a cloud symbol with a tick
+3) Login to Pupil Cloud to view or download your recording/s!
+
+### Local transfer
+1) Connect your OnePlus device to a PC via USB (using the USB cable supplied)
+2) Slide down from the top of the device's home-screen and click on 'Android System.USB charging this device'
```
Comment (papr) [inline src/invisible/user-guide/invisible-companion-app.md:95]:
```suggestion
2) Slide down from the top of the device's home-screen and click on 'Android System - USB charging this device'
```

File: src/invisible/user-guide/invisible-companion-app.md [hunk 2]
```diff
@@ -77,3 +80,20 @@ If more that one device is used and the data is required to be synchronised, mak
...
+3) Click on 'Tap for more options'
+4) Select 'Transfer Files'
+5) Open File Browser on your PC and access the Internal shared storage of your OnePlus device
+6) Pupil Invisible recordings can be found in the Pupil Invisible folder
```
Comment (papr) [inline src/invisible/user-guide/invisible-companion-app.md:99]:
```suggestion
6) Pupil Invisible recordings can be found in the Pupil Invisible folder

::: tip
On macOS, you require the <a href="https://www.android.com/filetransfer/" alt="Android File Transfer website">Android File Transfer</a> to browse and transfer files between your Mac computer and your Android device.
:::
```

**Review by papr — APPROVED** [review]
Two small suggestions at the bottom. Remaining documentation reads great!

File: src/invisible/user-guide/invisible-companion-app.md (hunk 2 above)
Comment (N-M-T) [reply src/invisible/user-guide/invisible-companion-app.md]: Thanks for that mac info!

Comment (N-M-T) [reply src/invisible/user-guide/invisible-companion-app.md]: Thanks for the mac info!

**Review by papr — APPROVED** [review]

File: src/invisible/README.md [hunk 3]
```diff
@@ -37,13 +37,19 @@ Always use the Connector Cable that came with your Pupil Invisible! If you are u
...
 
 If you run into any problems check out [troubleshooting](/invisible/user-guide/troubleshooting).
 
 <v-divider></v-divider>
 
+## 5. Playback and download
+
+You can play back any recording you have made in recordings view. 
+
+You can also playback and download in [Pupil Cloud](/cloud "Pupil Cloud documentation"),
```
Comment (willpatera) [inline src/invisible/README.md:50]:
```suggestion
After uploading to Pupil Cloud, you can playback recordings, enrich data, and download. Check out [Pupil Cloud](https://cloud.pupil-labs.com/ "Pupil Cloud") for more information. 
```

File: src/invisible/README.md [hunk 4]
```diff
@@ -37,13 +37,19 @@ Always use the Connector Cable that came with your Pupil Invisible! If you are u
...
 If you run into any problems check out [troubleshooting](/invisible/user-guide/troubleshooting).
 
 <v-divider></v-divider>
 
+## 5. Playback and download
+
+You can play back any recording you have made in recordings view. 
+
+You can also playback and download in [Pupil Cloud](/cloud "Pupil Cloud documentation"), 
+or [transfer directly](/invisible/user-guide/invisible-companion-app/#recording-transfer) to your
```
Comment (willpatera) [inline src/invisible/README.md:51]:
```suggestion

Alternatively you can [directly transfer recordings](/invisible/user-guide/invisible-companion-app/#recording-transfer) to your 
```

File: src/invisible/user-guide/invisible-companion-app.md [hunk 5]
```diff
@@ -77,3 +80,24 @@ If more that one device is used and the data is required to be synchronised, mak
 Pupil Monitor will warn you if a device is not in sync.
 
 More info on the technical implementation and quality of NTP synchronisation can be found in the [Pupil Invisible developer docs](/developer/invisible/#time-synchronization "Pupil Invisible developer docs - time synchronization").
+
+## Recording transfer
+### Pupil Cloud
+
+Invisible recordings are uploaded to the cloud from the Invisible Companion App to ensure stability, robustness and security. If you still have the recordings on the Companion device, follow these instructions:
```
Comment (willpatera) [inline src/invisible/user-guide/invisible-companion-app.md:87]:
```suggestion
Pupil Invisible recordings can be uploaded cloud storage directly from the Invisible Companion App. We recommend using Pupil Cloud for ease of user experience, stability, robustness, and data security. 

If you made recordings on Pupil Invisible Companion while cloud upload feature was disabled, you can re-enable and upload recordings to Cloud:```

File: src/invisible/user-guide/invisible-companion-app.md [hunk 6]
```diff
@@ -77,3 +80,24 @@ If more that one device is used and the data is required to be synchronised, mak
...
+
+1) Enable 'Cloud Upload' in the Companion App settings
+2) Wait until your recording/s are uploaded to Pupil Cloud. In the recording view, uploaded recordings will show a cloud symbol with a tick
+3) Login to Pupil Cloud to view or download your recording/s!
```
Comment (willpatera) [inline src/invisible/user-guide/invisible-companion-app.md:91]:
```suggestion
3) Login to [Pupil Cloud](https://cloud.pupil-labs.com "Pupil Cloud") to playback recordings, enrich data, and download.
```

**Review by willpatera — CHANGES_REQUESTED** [review]
@N-M-T looks good, I have left a few suggestions for you to consider.

**Pull request merged**